templates/gunicorn.py

```diff
@@ -0,0 +1,12 @@
+"""Add a worker for each available CPU.
+
+Most requests are handled quickly, so 1 worker should be able to handle many
+requests per second.
+"""
+
+workers = 1
```

Comment:
Might want to template this for admins who get a little more tiaas traffic?

Reply:
agreed, making it a variable makes sense, 1's a good default. (but I suspect no one is getting that much tiaas traffic. I'd be curious to know, maybe some day we add a `/metrics` endpoint for prometheus.)

Reply:
edit: https://github.com/galaxyproject/tiaas2/pull/48/ added it since it'll be nice to have.